src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -354,26 +414,50 @@ def _modify_job_updates(self, req: JobRequest) -> dict:
 
     def _cancel_jobs(self, req: JobRequest) -> dict:
         """
-        This cancels a running job.
-        If there are no valid jobs, this raises a JobRequestException.
-        If there's an error while attempting to cancel, this raises a NarrativeError.
-        In the end, after a successful cancel, this finishes up by fetching and returning the
-        job state with the new status.
+        Cancel a job or list of jobs. After sending the cancellation request, the job states
+        are refreshed and their new output states returned.
+
+        See JobManager.cancel_jobs() for more details.
+
+        :param req: job request containing job ID or list of job IDs to be cancelled
+        :type req: JobRequest
+
+        :return: job output states, indexed by job ID
+        :rtype: dict
         """
         job_id_list = self._get_job_ids(req)
         cancel_results = self._jm.cancel_jobs(job_id_list)
         self.send_comm_message(MESSAGE_TYPE["STATUS"], cancel_results)
```

Comment:
In `jm.send_comm_message`, all `STATUS` responses will have `last_checked` added in downstream